src/graphql/public/schema.graphql

```diff
@@ -724,7 +724,7 @@ type Mutation {
   userEmailRegistrationValidate(input: UserEmailRegistrationValidateInput!): UserEmailRegistrationValidatePayload!
   userLogin(input: UserLoginInput!): AuthTokenPayload!
   userLoginUpgrade(input: UserLoginUpgradeInput!): UpgradePayload!
-  userLogout: SuccessPayload!
+  userLogout(input: UserLogoutInput!): SuccessPayload!
```

Comment:
if input is required it will be a breaking change

Reply:
> if input is required it will be a breaking change. is this mutation not implemented in mobile/web wallet?

this was not used so we're good